common/src/View/FaceAttribsEditor.cpp

```diff
@@ -283,18 +355,26 @@ namespace TrenchBroom {
             m_surfaceValueEditor->setRange(min, max);
             m_surfaceValueEditor->setIncrements(1.0, 10.0, 100.0);
             m_surfaceValueEditor->setDigits(0, 6);
+            m_surfaceValueUnsetButton = createBitmapButton("Unset.svg", tr("Unset surface value"));
```

Comment:
I think that it's more fitting to use the "ResetTexture" icon here, too. Effectively, you are resetting these values to whatever is the default from the texture.

Reply:
You mean this one?

![Capture](https://user-images.githubusercontent.com/239161/130290906-bf668b9b-7e52-43ba-a093-894fb09f2d50.PNG)

I would rather use the specific icon I am using now because it's an idiom from Unreal.

That X button just resets texture alignment, leaving surface / content flags alone. So it's unrelated to what my new Unset buttons are doing and would be confusing to overload.

Reply:
I think of these icons in more general terms like `+` is for adding, `-` for deleting, `X` is for resetting, etc. The reason why I suggested `X` for this is that it also resets to defaults, in a sense. That one of them applies to UVs and the other applies to surface flags and such should be apparent from the placement of the buttons. So in that sense, we're not overloading this icon with a new meaning.

We originally used the curved arrow icon for this in the UV editor, but then changed it because we needed an icon for "Reset to world" and couldn't make that work with the curved arrow. In my view, if we now introduce the curved arrow again for something that also means "reset", then that's inconsistent, and I'd rather like to avoid that.

Reply:
OK, I see what you're saying. There's a subtle difference which is what I was trying to hint at with the different icons:

- the face/contents/value one is "Unset (and inherit the value from the parent [the texture], including future changes if the parent changes)"
- the texture axis one is "Set texture alignment to sensible defaults, based on the current face normal". i.e. there's no concept of inheritance here.

but for the sake of icons maybe these are close enough that we can call them both "Reset"

Reply:
Thanks! I agree that there is a slight semantic difference, but I believe that from the users' point of view, it doesn't matter as much.